miqa/settings.py

```diff
@@ -40,8 +40,17 @@ def before_binding(configuration: ComposedConfiguration) -> None:
         # Install additional apps
         configuration.INSTALLED_APPS += [
             's3_file_field',
+            'guardian',
         ]
 
+        # guardian's authentication backend
+        configuration.AUTHENTICATION_BACKENDS += [
+            'guardian.backends.ObjectPermissionBackend',
```

Comment:
What does this do?

Reply:
Part of the configuration for guardian, I was just following [the guide in the docs](https://django-guardian.readthedocs.io/en/stable/configuration.html#:~:text=AUTHENTICATION_BACKENDS%20%3D%20(%0A%20%20%20%20%27django.contrib.auth.backends.ModelBackend%27%2C%20%23%20this%20is%20default%0A%20%20%20%20%27guardian.backends.ObjectPermissionBackend%27%2C%0A))